bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import textwrap
+from collections import ChainMap
+from typing import Union
+
+from aiohttp.client_exceptions import ClientResponseError
+from discord import Color, Embed, Member, User
+from discord.ext.commands import Context, group
+
+from bot.constants import Channels, Guild, Roles, Webhooks
+from bot.decorators import with_role
+from bot.pagination import LinePaginator
+from .watchchannel import WatchChannel, proxy_user
+
+log = logging.getLogger(__name__)
+STAFF_ROLES = Roles.owner, Roles.admin, Roles.moderator, Roles.helpers    # <- In constants after the merge?
+
+
+class TalentPool(WatchChannel):
+    """Relays messages of helper candidates to the talent-pool channel to observe them."""
+
+    def __init__(self, bot) -> None:
+        super().__init__(
+            bot,
+            destination=Channels.talent_pool,
+            webhook_id=Webhooks.talent_pool,
+            api_endpoint='bot/nominations',
+            api_default_params={'active': 'true', 'ordering': '-inserted_at'},
+            logger=log,
+        )
+
+    @group(name='talentpool', aliases=('tp', 'talent', 'nomination', 'n'), invoke_without_command=True)
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def nomination_group(self, ctx: Context) -> None:
+        """Highlights the activity of helper nominees by relaying their messages to TalentPool."""
+
+        await ctx.invoke(self.bot.get_command("help"), "talentpool")
+
+    @nomination_group.command(name='watched', aliases=('all', 'list'))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def watched_command(self, ctx: Context, update_cache: bool = True) -> None:
+        """
+        Shows the users that are currently being monitored in TalentPool.
+
+        The optional kwarg `update_cache` can be used to update the user
+        cache using the API before listing the users.
+        """
+        await self.list_watched_users(ctx, update_cache)
+
+    @nomination_group.command(name='watch', aliases=('w', 'add', 'a'))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def watch_command(self, ctx: Context, user: Union[Member, User, proxy_user], *, reason: str) -> None:
+        """
+        Relay messages sent by the given `user` to the `#talent-pool` channel.
+
+        A `reason` for adding the user to TalentPool is required and will displayed
+        in the header when relaying messages of this user to the watchchannel.
```

Comment:
```suggestion
        in the header when relaying messages of this user to the channel.
```

Reply:
Is it really wrong to use a closed compound for a compound of two nouns? I mean, we're not talking about any channel, but a watchchannel specifically.

Reply:
The summary already mentions the talent pool channel so it should be clear by this point. Was also inconsistent with it being called `talent pool channel` in other instances but `watch channel` here.